src/event/mod.rs

```diff
@@ -23,13 +23,15 @@ pub use types::{TryIntoTime, TryIntoUrl};
 mod v03;
 
 pub use v03::Attributes as AttributesV03;
+pub use v03::AttributesIntoIterator as AttributesIntoIteratorV03;
```

Comment:
I think we don't need to make it public, just `pub(crate)` should be fine i think